src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -354,21 +354,28 @@ public function filter_add_body_classes( $classes ) {
 	 * @param string $file      Complete path to include the PHP File
 	 * @param array  $name      Template name
 	 * @param object $template  Instance of the Tribe__Template
-	 * 
+	 *
 	 * @return string
 	 */
 	public function filter_template_file( $file, $name, $template ) {
-		
-		if ( 
+		if (
 			! is_singular( TEC::POSTTYPE )
 			&& 'single-event' !== tribe_context()->get( 'view' )
 		) {
 			return $file;
 		}
-		
-		$override = V1_Event_Templates::getTemplateHierarchy( 'default-template' );
-		
-		return ! empty( $override ) ? $override : $file;
+
+		$theme_file = locate_template( 'tribe-events/default-template.php', false, false );
+
+		if ( ! $theme_file ) {
+			return $file;
+		}
+
+		if ( ! tribe_is_showing_all() && tribe_is_past_event() ) {
+			Tribe__Notices::set_notice( 'event-past', sprintf( esc_html__( 'This %s has passed.', 'the-events-calendar' ), tribe_get_event_label_singular_lowercase() ) );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 170 characters

<sub>hash: d7641050a2a1e988a8d6159fdfe365e5</sub>